chainregistry.go

```diff
@@ -173,6 +173,16 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 		bitcoindConn *chain.BitcoindClient
 	)
 
+	// Initialize the height hint cache within the chain directory.
+	chainDBPath := filepath.Join(
+		homeChainConfig.ChainDir, normalizeNetwork(activeNetParams.Name),
```

Comment:
nit: extract the result of `normalizeNetwork(activeNetParams.Name)` into variable if used elsewhere?

Reply:
Fixed.